etest/__init__.py

```diff
@@ -12,22 +12,22 @@
 
 import click
 
-from etest import docker, information, tests
+from etest import docker, information, qemu, tests
 
 logger = logging.getLogger(__name__)
 logger.propagate = False
 logger.addHandler(logging.NullHandler())
 
 
-def echo_check(check):
+def echo_check(check: tests.Test):
     """Print a check status."""
     if check.failed:
         click.secho("F", nl=False, fg="red")
     else:
-        click.secho("·", nl=False, fg="green")
+        click.secho("OK", nl=False, fg="green")
```

Comment:
The intention of this was to be a short version.  It'll be weird seeing:

OKFFFFOKFFFOKFFF

Reply:
Thats ok. Maybe we can use O?

Reply:
I was following the nosetest and pytest conventions of F and . why change that?

Reply:
I found it a bit confusing. If its a convention its ok imo, Ill change it back.

Reply:
Yeah, if I recall, the default is just dots but with -v you get more stuff.

Reply:
I want to ultimately change this for a progress bar in a future commit anyways

Reply:
Oh man, that'll get interesting because you have to calculate the test cases before starting runs.  I opted for this format to avoid that and start getting results as quickly as possible.

Reply:
It shouldnt be too hard. We just need to calculate how many use flag combinations there are, so n! for n use flags, and set that number of steps for non-verbose. For verbose, we just need to spawn a progress bar for each test with 5 steps, one per command.